lib/rules/require-dollar-before-jquery-assignment.js

```diff
@@ -75,6 +75,41 @@ module.exports.prototype = {
             var varName;
             var right;
 
+            function checkIfVarNameShouldStartWithDollar(left, varName, right) {
```

Comment:
Why is var not first in the list? The function name talks about var, not left or right